docs/contributor/02-10-operations.md

```diff
@@ -26,77 +26,74 @@ the prerequisites yourself.
 reconciliation proceeds.
-Otherwise, it is given the `Error` state with the condition reason `OlderCRExists` and message containing details
-about the CR responsible for reconciling the operand.
-
-Next, the reconciler looks for a `sap-btp-manager` Secret in the `kyma-system` Namespace with the label `app.kubernetes.io/managed-by: kcp-kyma-environment-broker`. This Secret contains the Service
-Manager credentials for SAP BTP Service Operator and should be delivered to the cluster by KEB. If the Secret is
-missing, an error is thrown, the reconciler sets `Warning` state (with the condition reason `MissingSecret`) in the CR and stops the reconciliation until the Secret
-is created. When the Secret is present in the cluster, the reconciler verifies whether it contains required data. The
+5. The reconciler looks for a `sap-btp-manager` Secret in the `kyma-system` Namespace with the label `app.kubernetes.io/managed-by: kcp-kyma-environment-broker`. This Secret contains the Service Manager credentials for SAP BTP Service Operator and should be delivered to the cluster by KEB. If the Secret is missing, an error is thrown (5a), and the reconciler sets the `Warning` state (with the condition reason `MissingSecret`) in the CR and stops the reconciliation until the Secret
+is created. 
+6. When the Secret is present in the cluster, the reconciler verifies whether it contains the required data. The
 Secret should contain the following keys: `clientid`, `clientsecret`, `sm_url`, `tokenurl`, `cluster_id`. None of the
-key values should be empty. If some required data is missing, the reconciler throws an error with the message about
-missing keys/values, sets the CR in `Error` state (reason `InvalidSecret`), and stops the reconciliation until there is a change in the required
+key values should be empty. If some required data is missing, the reconciler throws an error (6a) with the message about
+missing keys/values, sets the CR in the `Error` state (reason `InvalidSecret`), and stops the reconciliation until there is a change in the required
 Secret.
-
-After checking the Secret, the reconciler proceeds to apply and delete operations of the [module resources](/module-resources).
-The `module-resources` directory is created by one of GitHub Actions and contains manifests for applying and deleting operations. See [workflows](04-10-workflows.md#auto-update-chart-and-resources) for more details.
-First, the reconciler deletes outdated module resources stored as manifests in [to-delete.yml](/module-resources/delete/to-delete.yml).
-When all outdated resources are deleted successfully, the reconciler prepares current resources from manifests in the [apply](/module-resources/apply) directory to be applied to the cluster.
+7. After checking the Secret, the reconciler applies and deletes operations of the [module resources](/module-resources).
```

Comment:
`the reconciler applies and deletes operations of the module resources` doesn't sound right. It performs apply and delete operations on the module resources.